test/integration/an_item_can_be_removed_from_cart_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.